.github/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'java'
+
+repositories {
+    mavenCentral()
+}
+dependencies {
+    testImplementation group: 'junit', name: 'junit', version: '4.12'
+    testImplementation group: 'xerces', name: 'xercesImpl', version: '2.12.0'
+    testImplementation group: 'org.xmlresolver', name: 'xmlresolver', version: '3.1.0'
+    testImplementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
+}
+test {
+    systemProperty "repository", project.getProperty("repository")
```

Comment:
We need these in order to pass variables from command line with `-P` to JUnit tests.